modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,243 +53,371 @@ and device ID 0x1015
+
+. Verify the operator installations:
+
+.. Switch to the *Catalog* → *Installed Operators* page.
+
+.. Ensure that *SR-IOV Network Operator* is listed in the
+*riov-network-operator* project with a *Status* of *InstallSucceeded*.
+
 +
-[source,yaml,subs="attributes"]
-----
-kind: DaemonSet
-apiVersion: apps/v1
-metadata:
-  name: sriov-device-plugin
-  namespace: openshift-sriov
-  annotations:
-    kubernetes.io/description: |
-      This daemon set launches the SR-IOV network device plugin on each node.
-spec:
-  selector:
-    matchLabels:
-      app: sriov-device-plugin
-  updateStrategy:
-    type: RollingUpdate
-  template:
-    metadata:
-      labels:
-        app: sriov-device-plugin
-        component: network
-        type: infra
-        openshift.io/component: network
-    spec:
-      hostNetwork: true
-      nodeSelector:
-        kubernetes.io/os: linux
-      tolerations:
-      - operator: Exists
-      serviceAccountName: sriov-device-plugin
-      containers:
-      - name: sriov-device-plugin
-        image: quay.io/openshift/{image-prefix}-sriov-network-device-plugin:v4.0.0
-        args:
-        - --log-level=10
-        securityContext:
-          privileged: true
-        volumeMounts:
-        - name: devicesock
-          mountPath: /var/lib/kubelet/
-          readOnly: false
-        - name: net
-          mountPath: /sys/class/net
-          readOnly: true
-      volumes:
-        - name: devicesock
-          hostPath:
-            path: /var/lib/kubelet/
-        - name: net
-          hostPath:
-            path: /sys/class/net
-----
-
-. Run the following command to create the `sriov-device-plugin` DaemonSet:
+[NOTE]
+====
+During installation an operator might display a *Failed* status. If the operator
+then installs with an *InstallSucceeded* message, you can safely ignore
+the *Failed* message.
+====
+
 +
-----
-oc create -f sriov-device-plugin.yaml
-----
+If the operator does not appear as installed, to troubleshoot further:
 
-. Create a YAML file for the `sriov-cni` DaemonSet with the following contents:
 +
-[source,yaml,subs="attributes"]
+* Switch to the *Catalog* → *Operator Management* page and inspect
+the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors
+under *Status*.
+* Switch to the *Workloads* → *Pods* page and check the logs in any Pods in the
+`sriov-network-operator` projects that are reporting issues.
+
+== Discover SR-IOV network devices
+
+After the SR-IOV network Operator has been install successfully. The operator
+will try to discover all the SR-IOV capable network devices on worker nodes.
+User can find those information from the `SriovNetworkNodeState` Custom 
+Resources, which are generated and updated by the operator automatically. 
+
+One CR is created for each worker node, and shares the same name as the node. In
+the interface list, you can the information of the network devices.  
```

Comment:
missing word at `you can <> the information`

Reply:
Fixed.